src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationKeyVaultOptions.cs

```diff
@@ -34,5 +37,15 @@ public AzureAppConfigurationKeyVaultOptions Register(SecretClient secretClient)
             SecretClients.Add(secretClient);
             return this;
         }
+
+        /// <summary>
+        /// Sets the callback to be invoked for resolving those key vault references that have no registered <see cref="SecretClient"/>.
```

Comment:
> Sets the callback to be invoked for resolving those key vault references that have no registered <see cref="SecretClient"/>. [](start = 11, length = 125)

This can be rephrased as follows to keep it consistent with the summary comments for other methods.

`Sets the callback used to resolve key vault references that have no registered <see cref="SecretClient"/>.`